client/src/Movies/MovieList.js

```diff
@@ -21,34 +26,42 @@ export default class MovieList extends Component {
   }
 
   render() {
+    console.log(this.state.movies);
     return (
       <div className="movie-list">
         {this.state.movies.map(movie => (
-          <MovieDetails key={movie.id} movie={movie} />
+          <MovieCard key={movie.id} movie={movie} saveMovie = {this.saveMovie}  />
```

Comment:
```suggestion
          <MovieCard key={movie.id} movie={movie} saveMovie = {this.props.addToSavedList(movie)}  />
```
this.saveMovie isn't defined in this component